snmp/tests/compose/data/3com-generic.snmprec

```diff
@@ -0,0 +1,4 @@
+1.3.6.1.2.1.1.1.0|4|3Com Device Desc
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.43.1.99999999
+1.3.6.1.2.1.1.5.0|4|3com.device.name
+1.3.6.1.2.1.2.1.0|2|4
```

Comment:
Note: we only test `snmp.ifNumber` metric (1.3.6.1.2.1.2.1.0) to verify that we are correctly importing `_generic-if.yaml`.